src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -169,10 +169,32 @@
             <tag name="api_platform.filter" />
         </service>
 
+        <service id="sylius.api.tax_rates_start_date_filter" parent="api_platform.doctrine.orm.date_filter" public="true">
```

Comment:
Do we have contract tests for these filters?

Reply:
I'm uncertain if we need them. It is the ApiPlatform's built-in feature, they're covered in Behat scenarios and in such case we should test all of them what give as a pretty many combinations.

Reply:
We agreed to not add contract tests for this filter.

Reply:
IMO there is no need for additional tests, especially that this functionality is covered by behat scenarios